tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -418,7 +375,7 @@ void ChangeNumberSequential(NumberSequential* p)
 
 bool IsCorrectNumberSequential(NumberSequential* p)
 {
-	if(p->i32 != -0x80000000 || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 || 
+	if(p->i32 != (-0x7fffffff - 1) || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 || 
```

Comment:
This is basically `INT_MIN` but I wasn't sure that the necessary header is included cross-platform so I used this instead, using `-2147483648` also results in a warning.